presto-docs/src/main/sphinx/security/built-in-system-access-control.rst

```diff
@@ -169,6 +171,55 @@ following rules:
 For group-based rules to match, users need to be assigned to groups by a
 :doc:`/develop/group-provider`.
 
+.. _schema_rules:
+
+Schema Rules
+------------
+
+These rules govern who is considered an owner of a schema. The user is granted
+the privileges based on the first matching rule read from top to bottom. If no
```

Comment:
The matching being done in order applies to all rules and should be explained in an earlier section.

Reply:
Explained after the list of rules ends and have removed the mention of the ordering from each individual rule to avoid repetition.